templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -168,7 +169,7 @@ Resources:
               fi
               if ! (yum install chef); then
                 for i in `seq 1 5`; do
-                  yum install -y libxcrypt-compat "https://packages.chef.io/files/stable/chef/14.15.6/el/7/chef-14.15.6-1.el7.x86_64.rpm" && break
+                  yum install -y libxcrypt-compat "https://packages.chef.io/files/stable/chef/18.4.12/el/7/chef-18.4.12-1.el7.x86_64.rpm" && break
```

Comment:
future state: may want a script (github action) that gives a pr when there is a new chef version out to remind us to update and stay above any cve's.